docs/releases.rst

```diff
@@ -4,6 +4,27 @@
     Releases
 ======================
 
+tmt-1.38.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each execution of ``tmt-report-result`` command inside a shell test will now
+create a tmt subresult. The parent result outcome is reduced from all
+subresults outcomes. If ``tmt-report-result`` is not called during the test,
+the shell test framework behavior remains the same - the test script exit code
+still has an impact on the parent test result. See also
+:ref:`/stories/features/report-result`.
+
+The beakerlib test framework tests now generate tmt subresults. The behavior is
+very similar to the shell test framework with ``tmt-report-result`` command
+calls (see above). The ``tmt-report-result`` now gets called with every
+``rlPhaseEnd`` macro and the tmt subresult gets created. The
+difference is that the subresults outcomes do not influence the parent result
+outcome in any way - the parent result outcome is always set to the outcome
+reported by the Beaker. The subresults are always assigned under the parent tmt
```

Comment:
This is and isn't true - subresults do not influence the parent result *as far as tmt is interested*, but beakerlib *does* count them in. A failed phase - subresult - will affect the whole test outcome as reported by beakerlib framework. How about something like this?

> The difference is that the subresults outcomes are not evaluated by tmt, tmt only captures them, and tmt relies on Beakerlib and its result reporting - which does take outcomes of phases into account - to determine the final test outcome. The subresults are always assigned under the parent tmt result...